pkg/exec/exec_test.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2018 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package exec
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestNewShellExec(t *testing.T) {
+	testCases := map[string]struct {
+		binary      string
+		expectValue *ShellExec
+	}{
+		"String Empty": {
+			binary:      "",
+			expectValue: &ShellExec{binary: ""},
+		},
+		"Present string value": {
+			binary:      "_MY_PRESENT_STRING_",
+			expectValue: &ShellExec{binary: "_MY_PRESENT_STRING_"},
+		},
+	}
+	for k, v := range testCases {
+		t.Run(k, func(t *testing.T) {
+
+			if !reflect.DeepEqual(v.binary, v.expectValue.binary) {
```

Comment:
reflect is not really required. string comparision should suffice.
```suggestion
			if v.binary != v.expectValue.binary {
```